client/asset/dcr/dcr.go

```diff
@@ -759,6 +788,39 @@ func (dcr *ExchangeWallet) feeRate(confTarget uint64) (uint64, error) {
 	return 1 + uint64(atomsPerKB)/1000, nil // dcrPerKB * 1e8 / 1e3
 }
 
+// externalFeeEstimator gets the fee rate from the external API
+func externalFeeEstimator(ctx context.Context, nb uint64) (float64, error) {
+	var url string
+	if dex.Testnet == 1 {
+		url = testnetExternalApiUrl
+	} else {
+		url = externalApiUrl
+	}
```

Comment:
`if dex.Testnet == 1` is always `true`. You'll need to work with the `dex.Network` passed to `NewWallet` somehow. You **could** inspect the `*chaincfg.Params` to figure out what network you're on, but using the `dex.Network` would be better.

Reply:
my bad. Now I am adding a new global var `isTestnet`, which is stored on `NewWallet` and check when calling the `externalFeeEstimator`